### ISSUE TITLE
MVV # 15

### ISSUE THREAD

**Issue by JuankBell** [TECommons/TokenLog-SoftGov#15]
### What is the vision of the Token Engineering Commons?

To be the nucleus for token engineering research, development and education. The Commons will support ethical, robust and resilient practices of token engineering for the benefit of humanity. 

### What is the mission of the Token Engineering Commons? (How are we going to get there?)

To steward efficient collection and distribution of funds to token engineering projects. Beyond financial incentives, the Token Engineering Commons fosters a nourishing environment for collaboration and value creation. 

### What are the core values of the Token Engineering Commons? 

The Token Engineering Commons acknowledges the social importance of economic systems. It places trust, constructive inquiry and gratitude as foundations for a healthy and active community. 
Members are encouraged to be radically open source, non-hierarchical, transparent and mutually accountable.
The Commons will strive to support token engineering projects that appreciate the value of safe, ethical, diverse and inclusive economic systems.

--- 

🗳 When you are finished reading the MVVs, head over to [TokenLog](https://tokenlog.xyz/CommonsBuild/TokenLog-SoftGov) and vote for your favorite vision!

**Comment by GriffGreen** [comment]
Updated the footer

**Comment by tamarandom** [comment]
I really like this mission but I think it leaves out the regenerative aspect that makes the TEC so unique!! "Collection and distribution" does not do justice to what we are doing! 

The values is spot on. It's great.

The vision could be more bold, pie-in-the-sky ambitious!

**Comment by danelsuga** [comment]
I agree with @tamarandom regarding the vision: it makes the TEC sound like a repository instead of an active creator. 

The mission also does not emphasize the significance of the cultural build: it uses the word "efficient", which could mean anything. The second sentence offers more clarity by "nourishing" and "value creation", but I think it needs more direct and strong language. 

Equally, the term "acknowledge" in the values feels too passive, but all of the information in the values is great!

**Comment by liviade1** [comment]
I agree with Katy and Tamara, it feels like the punch lines are missing. I like how "Token Engineering Commons fosters a nourishing environment for collaboration and value creation" brings the image of a very inspiring place where things happen. I think the vision could incorporate more this sentiment.

**Comment by laurenluz** [comment]
I really like the vision in this one! For me it is the more understandable, direct and practical. It covers both the technical and the human element... kind of this repetition of the idea that good engineers build for the good of all of humanity.

I love the value too, just perhaps it could be shortened a little for better flow.
